src/main/scala/com/agar/core/region/Region.scala

```diff
@@ -80,11 +120,12 @@ class Region(journal: ActorRef)(width: Int, height: Int) extends Actor with Stas
       }.toMap
   }
 
-  private def createNewPlayer(position: Vector2d): ActorRef = {
-    context.actorOf(Player.props(position, WEIGHT_AT_START))
+  private def createNewPlayer(position: Vector2d, weight: Int): ActorRef = {
+    context.actorOf(Player.props(position, weight), s"player-${UUID.randomUUID().toString}")
```

Comment:
Why we need a name ? ActorRef is not enough ?

Reply:
I agree. This is related to the (in)famous actor access using UUID expected in the first version.